src/wallet/wallet.h

```diff
@@ -677,6 +678,9 @@ class CWallet final : public CCryptoKeyStore, private interfaces::Chain::Notific
      * Should be called with non-zero block_hash and posInBlock if this is for a transaction that is included in a block. */
     void SyncTransaction(const CTransactionRef& tx, const uint256& block_hash, int posInBlock = 0, bool update_tx = true) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
 
+    /** Return all scriptPubKeys to be used in a BIP 157 block filter */
```

Comment:
I'd note in the comment that it's a basic (type) block filter (in cases an other types are added).

Reply:
Mentioned "basic"